sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/OpenAIClientTestBase.java

```diff
@@ -47,6 +49,7 @@ public abstract class OpenAIClientTestBase extends TestProxyTestBase {
     OpenAIClientBuilder getOpenAIClientBuilder(HttpClient httpClient, OpenAIServiceVersion serviceVersion) {
         OpenAIClientBuilder builder = new OpenAIClientBuilder()
             .httpClient(httpClient)
+            .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
```

Comment:
oh Interesting, does this mean that the JSON for the requests/reponses will start showing up in standard output for tests?

Reply:
It is for debug purpose. I will remove it.